scripts/reset-test-db.sh

```diff
@@ -22,6 +22,8 @@
 #
 ############################################################################
 
+set -e
```

Comment:
I put this in as a safety measure because:

- I had added [the private settings file described here](https://github.com/edx/edx-platform/wiki/Developing-on-the-edX-Developer-Stack#making-the-local-servers-run-faster) to speed up my devstack
- That private settings file overrides the `INSTALLED_APPS` changes in the bok choy settings file, which means that `django_extensions` is no longer in there
- That causes this script to fail, becase the `reset_db` command is no longer available
- But the script would just ignore the failure and carry on as per bash's default behaviour, causing much confusion